core/src/scene/filters.h

```diff
@@ -57,19 +62,29 @@ struct Filter {
     Filter() : data(none_type{}) {}
     Filter(Data _data) : data(std::move(_data)) {}
 
+    static std::vector<Filter> sort(const std::vector<Filter>& filters);
+    void print(int _indent = 0) const;
+    int matchCost() const;
```

Comment:
This name might be confusing with all the other "matchX" methods. `getCost()`? `cost()`?